### PR TITLE
Add code of conduct (contributor covenant)

### PR DESCRIPTION
**Pull request by Powersource**
This is a proposal by @keks , it is not voted on so far

### CODE REVIEW

File: code-of-conduct.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (Powersource) [inline code-of-conduct.md:63]: todo

**Comment by Powersource** [comment]
Source https://www.contributor-covenant.org/version/2/0/code_of_conduct/

**Comment by arj03** [comment]
I'm a bit wary about the use of community leaders, but otherwise I think it looks good. Might be a good idea to have a subteam responsible for this.

**Comment by cblgh** [comment]
+1 on arj on wariness about the term 'community leaders'

**Comment by cblgh** [comment]
as an alternative to the default style of code of conduct proposed in this PR, i really like this one by a creative space in rotterdam: http://varia.zone/en/pages/code-of-conduct.html
edit: backup of varia zone's COC https://cblgh.org/dl/Code%20of%20Conduct.html

it reads less like aggressive legalese and more like a human & personal document reflecting expectations and setting of boundaries.

having read varia zone's version of such a document (or recurse center's [code of conduct](https://www.recurse.com/code-of-conduct) and [social rules](https://www.recurse.com/social-rules)) i quite prefer the latter style, rather than the one proposed in this PR.

---

after a brief moment of reflection, i think i am missing an introduction / context-setting section in the contributor covenant, or the **why** section that places it into the actual context of the consortium. 

incorporating the proposed style of legalese as a founding & culture-setting document feels like a departure from the community aspect of ssb.

**Comment by ZELFs** [comment]
+1 on both arj and cblghs points

The code of conduct you linked @cblgh 404d for me tho :o

**Comment by cblgh** [comment]
the original varia.zone code of conduct link is now working again:)

**Comment by cblgh** [comment]
from today's meeting: keks is also included in taking on this, but i committed to getting the ball rolling on this PR again

**Comment by elavoie** [comment]
We don't necessarily need community leaders to enforce this. One alternative is for vouchers to intervene first through private or public discussions with the person, and second by removing their vouches if the person keeps behaving this way. 

So this line of reasoning would suggest each voucher is responsible for making a judgement on wether the persons they are vouching for behaves according to the community agreed principles. 

If none of the immediate vouchers of a person agree to do this work, then the escalation would need to happen on those that vouch for the vouchers. If that happens however, I would believe the situation of harassment or abuse is sufficiently ambiguous to require a larger community introspection and resolution.

**Comment by Powersource** [comment]
Closing this since we have a new PR for this https://github.com/scuttlebutt-eu/important-documents/pull/41

**Pull request closed without merging**